criu/proc_parse.c

```diff
@@ -842,6 +842,7 @@ int parse_smaps(pid_t pid, struct vm_area_list *vma_area_list, dump_filemap_t du
 			goto err;
 		}
 
+		pr_info("Handling VMA with the following smaps entry: %s\n", str);
```

Comment:
I don't think that pr_info is a good choice in this case.  smaps can contain a lot of entries, but printing log messages isn't so fast, so it can effect performance.